resources/views/student-council/economic-committee/app.blade.php

```diff
@@ -21,32 +21,27 @@
                 </blockquote>
                 @can('handleAny', \App\Models\Checkout::class)
                 <div class="row">
-                    <div class="col s12 m12 l6 xl3" style="margin-bottom:5px">
+                    <div class="col s12 m12 l6 xl6" style="margin-bottom:5px">
                         <a href="{{ route('kktnetreg') }}" class="btn waves-effect" style="width:100%">
                             @lang('checkout.pay_kktnetreg')</a>
                     </div>
-                    <div class="col s12 m12 l6 xl3" style="margin-bottom:5px">
-                        <a href="#" class="btn waves-effect" disabled style="width:100%">
-                            @lang('checkout.transaction_for_workshop')</a>
-                    </div>
-                    <div class="col s12 m12 l6 xl3" style="margin-bottom:5px">
+                    <div class="col s12 m12 l6 xl6" style="margin-bottom:5px">
                         <a href="{{ route('economic_committee.transaction') }}" class="btn waves-effect" style="width:100%">
                             @lang('checkout.other_transaction')</a>
                     </div>
-                    <div class="col s12 m12 l6 xl3" style="margin-bottom:5px">
-                        <a href="#" class="btn waves-effect" disabled style="width:100%">
-                            @lang('checkout.receipts')</a>
-                    </div>
                 </div>
                 @endcan
             </div>
         </div>
     </div>
-    @foreach($data as $semester => $row)
+    @foreach($data as $row)
+    @php
+    $semester = $row['semester'];
+    @endphp
     <div class="col s12">
         <div class="card">
             <div class="card-content">
-                <span class="card-title">{{ $semester }}</span>
+                <span class="card-title">{{ $semester->tag().' ('.$semester->getStartDate()->format('Y.m.d').'-'.$semester->getEndDate()->format('Y.m.d').')'}}</span>
```

Comment:
This date should be added as a getter to the Semeter model. (BTW, is it necessary to show it there?)